test/CMakeLists.txt

```diff
@@ -705,17 +703,17 @@ set(IMPLICITGEMM_MLIR_ENV_F ${IMPLICITGEMM_MLIR_ENV_BASE} MIOPEN_DEBUG_FIND_ONLY
 set(IMPLICITGEMM_MLIR_ENV_B ${IMPLICITGEMM_MLIR_ENV_BASE} MIOPEN_DEBUG_FIND_ONLY_SOLVER=ConvMlirIgemmBwd)
 set(IMPLICITGEMM_MLIR_ENV_W ${IMPLICITGEMM_MLIR_ENV_BASE} MIOPEN_DEBUG_FIND_ONLY_SOLVER=ConvMlirIgemmWrW)
 
-set(IMPLICITGEMM_MLIR_ARGS_F ${IMPLICITGEMM_ARGS} --verbose --disable-backward-data --disable-backward-weights)
-set(IMPLICITGEMM_MLIR_ARGS_B ${IMPLICITGEMM_ARGS} --verbose --disable-forward --disable-backward-weights)
-set(IMPLICITGEMM_MLIR_ARGS_W ${IMPLICITGEMM_ARGS} --verbose --disable-forward --disable-backward-data)
+set(TEST_CONV_VERBOSE_F ${MIOPEN_TEST_FLOAT_ARG} --verbose --disable-backward-data --disable-backward-weights)
```

Comment:
IMPLICITGEMM_MLIR_ARGS_F is not MLIR specific, but `test_conv` specific thing.

Reply:
> IMPLICITGEMM_MLIR_ARGS_F is not MLIR specific, but `test_conv` specific thing.

@atamazov may be a naive question. Not quite clear to me why Test_Conv only do forward direction, i.e.  IMPLICITGEMM_MLIR_ARGS_B  and IMPLICITGEMM_MLIR_ARGS_W are removed, could you give me some idea?

Reply:
test_conv does all directions by default. TEST_CONV_VERBOSE_F injects options that disables all directions except forward when we need to keep only forward. For example `MIOPEN_DEBUG_FIND_ONLY_SOLVER=ConvMlirIgemmFwd` allows only Forward MLIR solver to work, so we disable Backward and WrW in test (otherwise these two would fail).

TEST_CONV_VERBOSE_B/W are used in similar cases. Did I answer your question?